app/server/appsmith-server/src/main/java/com/appsmith/server/solutions/EmailEventHandler.java

```diff
@@ -99,6 +98,16 @@ public void handle(CommentThreadClosedEvent event) {
         .subscribe();
     }
 
+    private String getCommentThreadLink(String applicationId, String pageId, String threadId, UserRole userRole, String originHeader) {
+        String urlPostfix = "/edit";
+        if(userRole.getRole().equals(AppsmithRole.ORGANIZATION_VIEWER)) {
```

Comment:
On the rare off chance that we'll have to deal with unnecessary NPE's, when using `.equals` with a constant, let's have the constant on the left. This way, even if `.getRole` starts returning `null` next year, it'll just be false and not throw an NPE.

```suggestion
        if (AppsmithRole.ORGANIZATION_VIEWER.equals(userRole.getRole()) {
```